sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/python/GpuWindowInPandasExecBase.scala

```diff
@@ -262,7 +262,6 @@ trait GpuWindowInPandasExecBase extends UnaryExecNode with GpuExec {
           function match {
             case GpuAggregateExpression(_, _, _, _, _) => collect("AGGREGATE", frame, e)
             case _: GpuAggregateWindowFunction => collect("AGGREGATE", frame, e)
-            case _: GpuPythonUDF => collect("AGGREGATE", frame, e)
```

Comment:
Similar situation here.  `GpuPythonUDF` derives from `GpuAggregateWindowFunction`, hence the unreachable code warning.  The implementation here is the same as the case that will be taken in practice, so this can simply be deleted.

Reply:
Because this is in a file that is specific to the pandas operations it might be nice to put a comment by line 263 just to say that this includes `GpuPythonUDF`